scale/docs/rest/v6/strike.rst

```diff
@@ -521,7 +521,7 @@ A strike configuration JSON describes a set of configuration settings that affec
 +----------------------------+----------------+----------+--------------------------------------------------------------------+
 | .type                      | String         | Required | The type of the monitor. Must be either 'dir-watcher' or 's3'      |
 +----------------------------+----------------+----------+--------------------------------------------------------------------+
-| .transfer_suffix           | String         | Required | (dir-watcher)Defines a suffix that is used on the file names to    |
+| .transfer_suffix           | String         | Optional | (dir-watcher)Defines a suffix that is used on the file names to    |
```

Comment:
I would document here that if it's not provided, then a default of '_tmp' will be used.